src/components/shareMenu/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Input, Button, Typography, message } from 'antd';
+import TwitterIcon from '../../assets/images/twitter-icon.png';
+import EmailIcon from '../../assets/images/email-icon.png';
+import FBIcon from '../../assets/images/facebook-icon.png';
+import CopyIcon from '../../assets/images/copy-icon.png';
+
+const ShareMenuContainer = styled.div`
+  max-width: 550px;
+  margin: 20px 0px;
+`;
+
+const StyledButton = styled(Button)`
```

Comment:
nit: `StyledButton` doesn't really convey the intent of how and where it should be used. Maybe we should rename it?

Reply:
yeah that's fair - renaming to MediaShareButton